public/js/core.js

```diff
@@ -25,9 +25,9 @@ app.config(function ($stateProvider, $urlRouterProvider) {
     $urlRouterProvider.otherwise('/');
 });
 
-app.controller('mainController', function ($scope, $state, Todo) {
+app.controller("mainController", function ($scope, $state, Todo) {
```

Comment:
In Javascript here's no differences of using "" and ''. So it's valid in both cases. For me it is unspoken rule - I use ' for strings like variables or just name of something and use " for some log messages or more long strings then one word string. But you can choose you way :)